tests/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2018 Red Hat, Inc.
+#
+
+FROM fedora:27
+
+MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
+
+ARG DOCKER_PREFIX=kubevirt
```

Comment:
I think that build ARGS are not the right decision.  "docker.io/kubevirt", "latest" and "kube-system" should be the default values for the tests binaries anyway (maybe check). Further these values depend on the installation on the specific cluster under test.